tpu-client-next/tests/connection_workers_scheduler_test.rs

```diff
@@ -583,16 +584,16 @@ async fn test_rate_limiting() {
     scheduler_cancel.cancel();
     let stats = join_scheduler(scheduler_handle).await;
 
-    // Accept both 0 and 1 as valid values for it as we may run into connection handshake timeout
-    // error within the TEST_MAX_TIME.
+    dbg!(&stats);
     assert!(
         stats
             == SendTransactionStatsNonAtomic {
-                connection_error_timed_out: 1,
+                successfully_sent: 2,
+                write_error_connection_lost: 2,
                 ..Default::default()
             }
-            || stats == SendTransactionStatsNonAtomic::default()
     );
+    //    drop(connection_to_reach_limit);
```

Comment:
superfluous commented code

Reply:
Fixed in https://github.com/anza-xyz/agave/pull/6937/commits/39b77f42d77d0da44fb3f2e240fdaf8b2d31aeaa thank you!